### PR TITLE
APM > .NET > Add links to samples in github repo

### PR DESCRIPTION
**Pull request by jaycdave88**
### What does this PR do?
Adding sample examples from `dd-trace-dotnet` to public docs.

### Motivation
Customer awareness 

### Preview
https://docs-staging.datadoghq.com/jay.dave/adding_github_links_to_public_docs/tracing/setup_overview/setup/dotnet-framework?tab=environmentvariables#further-reading

https://docs-staging.datadoghq.com/jay.dave/adding_github_links_to_public_docs/tracing/setup_overview/setup/dotnet-framework?tab=environmentvariables#further-reading

https://docs-staging.datadoghq.com/jay.dave/adding_github_links_to_public_docs/tracing/setup_overview/compatibility_requirements/dotnet-core#further-reading

https://docs-staging.datadoghq.com/jay.dave/adding_github_links_to_public_docs/tracing/setup_overview/compatibility_requirements/dotnet-framework#further-reading

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -429,8 +432,8 @@ The following table lists configuration variables that are available only when u
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /tracing/compatibility_requirements/dotnet-core
-[2]: https://app.datadoghq.com/apm/docs
+[1]: https://app.datadoghq.com/apm/docs
+[2]: /tracing/compatibility_requirements/dotnet-core
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:436]:
```suggestion
[1]: /tracing/setup_overview/compatibility_requirements/dotnet-framework
[2]: /tracing/setup_overview/compatibility_requirements/dotnet-core
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 2]
```diff
@@ -419,7 +422,7 @@ The following table lists configuration variables that are available only when u
...
+The following table lists configuration variables that are available only when using automatic instrumentation and can be set for each integration. Use the first name (e.g. `DD_<INTEGRATION>_ENABLED`) when setting environment variables or configuration files. The second name (e.g. `Enabled`), indicates the name the `IntegrationSettings` property to use when changing settings in the code. Access these properties through the `TracerSettings.Integrations[]` indexer. Integration names are listed in the [Integrations][2] section. **Note:** On Linux, the names of environment variables are case-sensitive.
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:425]: I cannot tell what the [Integrations][?] link is supposed to be pointing at. Do you know?

**Review by kayayarai — CHANGES_REQUESTED** [review]
There are some link numbering problems in setup_overview/setup/dotnet-core.md. Please check each link in that page. (Links on tabs should stay on tabs.)

File: content/en/tracing/setup_overview/setup/dotnet-core.md (hunk 2 above)
Comment (andrewardito) [reply content/en/tracing/setup_overview/setup/dotnet-core.md:425]: This is probably a holdover from when we split compatibility reqs out as a separate page - the specific link would be this: /tracing/setup_overview/compatibility_requirements/dotnet-core/#integrations , but I think just linking to the compatibility page is fine.

File: content/en/tracing/setup_overview/setup/dotnet-core.md (hunk 1 above)
Comment (andrewardito) [reply content/en/tracing/setup_overview/setup/dotnet-core.md:436]: on the dotnet-core page we should only be linking to [2], so all [1]'s and [2]'s should become [2] and link to dotnet-core

**Review by andrewardito — APPROVED** [review]
Links should be working now.

**Review by kayayarai — APPROVED** [review]

**Pull request merged**